docs/serial_console.md

```diff
@@ -0,0 +1,84 @@
+Serial console output in coreboot
+=================================
+
+Along with v4.8.0.6 we introduced possibility to enable COM2 as the main serial
+port for serial console output in coreboot and SeaBIOS. Because this is not
+enabled in runtime configuration, separate binary must be built in order to get
+the output on COM2.
+
+## Building coreboot firmware with console on COM2
+
+Building image capable of printing output on COM2 is relatively easy.
+
+### Users guide
+
+To build the firmware from scratch and change the output to COM2 follow the
+steps:
+
+1. Clone the [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
+2. Pull or [build](https://github.com/pcengines/pce-fw-builder#building-docker-image)
+  docker container:
+
+  ```
+  docker pull pcengines/pce-fw-builder
+  ```
+
+  > COM2 is supported only on mainline releases for now, so legacy container is
+  > not needed
+
+3. Build v4.8.0.6 image:
+
+  ```
+  ./build.sh release v4.8.0.6 apux
```

Comment:
@miczyg1 perhaps an info about changing `apux` to appropriate platform is required.

Reply:
@krystian-hebel done